packages/large-response-middleware/src/index.ts

```diff
@@ -101,6 +103,16 @@ export const withLargeResponseHandler = ({
               response_size_mb: contentLengthMB.toFixed(2),
               $payload_ref,
             });
+
+            response.isBase64Encoded = false;
+            response.statusCode = 413;
+            response.body = JSON.stringify({
+              message: customErrorMessage
+                ? typeof customErrorMessage === 'string'
+                  ? customErrorMessage
+                  : customErrorMessage(event)
+                : LARGE_RESPONSE_USER_INFO,
+            });
```

Comment:
I would refactor a bit to ease the reading.

Plus we should protect that function call and fallback to whatever comes on customErrorMessage, if it's not a string or function.

```suggestion
            response.isBase64Encoded = false;
            response.statusCode = 413;
            const responseErrorMessage =  customErrorMessage ?? LARGE_RESPONSE_USER_INFO;
            
            response.body = JSON.stringify({
              message: typeof responseErrorMessage === 'string'
                  ? responseErrorMessage
                  : customErrorMessage?.(event) || customErrorMessage,
            });
```

Reply:
Thanks @jpinho! did a change based on your fallback approach suggestion, tks! 🙌